src/About.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
```

Comment:
Make sure each component has the correct name. In other words this one should be About not App. Then down the bottom we should have export default About.